### PR TITLE
Add notes for 04-15-2019 TSC meeting

### PR DESCRIPTION
**Pull request by carl-foundry**

### CODE REVIEW

File: docs/tsc/meetings/2019-04-15.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+Host: Michael Dolan
+
+Rotating Secretary: Carl Rand
+
+Attendees:
+  * [ ] Mark Boorer (_TSC_) - Industrial Light & Magic
+  * [ ] Sean Cooper (_TSC_) - DNEG
+  * [X] Michael Dolan (_TSC Chair_) - Sony Pictures Imageworks
+  * [X] Larry Gritz (_TSC_) - Sony Pictures Imageworks
+  * [ ] Thanh Ha - Academy Software Foundation / Linux Foundation
```
Comment (hodoulp) [inline docs/tsc/meetings/2019-04-15.md]: As Thanh is not anymore working for the Linux Foundation, his name could be removed from the list.

File: docs/tsc/meetings/2019-04-15.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  * [X] Patrick Hodoul (_TSC_) - Autodesk
+  * [ ] John Mertic - Academy Software Foundation / Linux Foundation
+  * [X] Carl Rand (_TSC_) - Foundry
+  * [X] Doug Walker (_TSC Chief Architect_) - Autodesk
+  * [X] Kevin Wheatley (_TSC_) - Framestore
+  * [X] Bernard Lefebvre
```
Comment (hodoulp) [inline docs/tsc/meetings/2019-04-15.md]: Bernard is from Autodesk.

File: docs/tsc/meetings/2019-04-15.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    - Mail address will be security.ocio.org
+    - Larry: Is a private mail address private enough?
+    - Michael: The private aspect is only sugested. Other sites don't tend to go beyond a private mail. And doing so would be quite a bit more work and far more complicated.
+    - Michael has tested the mail system by emailing into it.
+    - Larry: We just need to make sure it's a mail that isn't publicly visible. It's very unlikely that someone is going to need it to be more private than that.
+    - Michael: The system states that it's private.
+    - John is currently setting up the email side of things.
+
+* Updates on ongoing tasks - Python bindings
+    - Michal is actively working on the bindings.
```
Comment (hodoulp) [inline docs/tsc/meetings/2019-04-15.md]: Several typos for the name -> Michael

File: docs/tsc/meetings/2019-04-15.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    - Michal: The current API is not very Pythonic.
+    - Michal: Do we need backwards comaptability? Or can we split it?
+    - Kevin: It will depend on when we need to support Python 3 and whether we need to do something sooner rather than later.
+    - Kevin: If this is going to be in version 1.x then we would need to keep backwards compatibility.
+    - Michal: In 2.x we will have to remove some things regardless.
+    - Michal: Pushing the API changes back to 1.x is something we could do.
+    - Patrick: In 1.x the C API shouldn't change. Should this then not be the case for Python API?
+    - Michal: The option is there to backport the work done in 2.x to 1.x.
+    - Doug: For the near term we need to maintain backwards comaptibility. Could we decide later and see what the new API will be like?
```
Comment (hodoulp) [inline docs/tsc/meetings/2019-04-15.md]: typo 'comaptibility' -> 'compatibility'

File: docs/tsc/meetings/2019-04-15.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+    - Michael: The private aspect is only sugested. Other sites don't tend to go beyond a private mail. And doing so would be quite a bit more work and far more complicated.
+    - Michael has tested the mail system by emailing into it.
+    - Larry: We just need to make sure it's a mail that isn't publicly visible. It's very unlikely that someone is going to need it to be more private than that.
+    - Michael: The system states that it's private.
+    - John is currently setting up the email side of things.
+
+* Updates on ongoing tasks - Python bindings
+    - Michal is actively working on the bindings.
+    - Michal: The current API is not very Pythonic.
+    - Michal: Do we need backwards comaptability? Or can we split it?
```
Comment (hodoulp) [inline docs/tsc/meetings/2019-04-15.md]: typo 'comaptability' -> 'compatibility'

File: docs/tsc/meetings/2019-04-15.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+    - Doug: For the near term we need to maintain backwards comaptibility. Could we decide later and see what the new API will be like?
+    - Michal is starting off now by writing the documentation.
+        - Trying to balance out the types to reduce the inconsistences.
+        - Currently maintaining backwards compatibility.
+    - Larry: Switching the bindings to PyBind 11 might be a good time.
+    - Carl: Could we mark parts of the API as deprecated and remove them later to give people warning?
+        - Michael: The problem with this is trying to figure out the deprecation mechanism to use.
+    - Michael: OCIO 2.x will support both Python 2 and 3
+    - Kevin: We should make the C and Python API change at the same time. This will make it easier for users.
+    - Larry: The C and Python API should only deviate in places where it's natural for the specific language.
```
Comment (hodoulp) [inline docs/tsc/meetings/2019-04-15.md]: [Minor] Technically OCIO library has a 'C++' (or C/C++) API.

File: docs/tsc/meetings/2019-04-15.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+  * [X] Doug Walker (_TSC Chief Architect_) - Autodesk
+  * [X] Kevin Wheatley (_TSC_) - Framestore
+  * [X] Bernard Lefebvre - Autodesk
+
+# **OCIO TSC Meeting Notes**
+
+* Vulnerability reporting
+    - Currently going through the CII badge discussions this week.
+    - Mainly focusing on documentation.
+    - Looking at offline private reporting mail of security vunerabilities.
```
Comment (KevinJW) [inline docs/tsc/meetings/2019-04-15.md]: vulnerabilities

File: docs/tsc/meetings/2019-04-15.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+    - Mail address will be security.ocio.org
+    - Larry: Is a private mail address private enough?
+    - Michael: The private aspect is only sugested. Other sites don't tend to go beyond a private mail. And doing so would be quite a bit more work and far more complicated.
```
Comment (KevinJW) [inline docs/tsc/meetings/2019-04-15.md]: suggested

**Pull request merged**